src/Quartz/Impl/JobDetailImpl.cs

```diff
@@ -140,6 +168,42 @@ public JobDetailImpl(string name, string group, Type jobType, bool isDurable, bo
             RequestsRecovery = requestsRecovery;
         }
 
+        /// <summary>
+        /// Create a <see cref="IJobDetail" /> with the given name, and group, and
+        /// the given settings of all the other properties.
+        /// </summary>
+        /// <param name="key">The key of the job.</param>
+        /// <param name="jobType">Type of the job.</param>
+        /// <param name="description">The description given to the <see cref="IJob" /> instance by its creator.</param>
+        /// <param name="isDurable">if set to <c>true</c>, job will be durable.</param>
+        /// <param name="requestsRecovery">if set to <c>true</c>, job will request recovery.</param>
+        /// <param name="jobDataMap">The data that is associated with the <see cref="IJob" />.</param>
+        /// <param name="disallowConcurrentExecution">Indicates whether or not concurrent exection of the job should be disallowed.</param>
+        /// <param name="persistJobDataAfterExecution">Indicates whether or not job data should re-stored when execution of the job completes.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="key"/> is <see langword="null"/>.</exception>
+        internal JobDetailImpl(JobKey key,
```

Comment:
nit: I prefer every arguments on new line when there are arguments on new lines, so this would have less indent too